tensorflow/compiler/xla/service/cpu/test_target_triple_helper.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+  
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_TEST_TARGET_TRIPLE_HELPER_H_
+#define TENSORFLOW_TEST_TARGET_TRIPLE_HELPER_H_
+
+#if (defined(__powerpc__) || defined(__ppc__) && (__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__))
```

Comment:
This seems a bit odd, why not use a ppc target for big endian PPC?  If this is intentional can you clarify the rationale as a comment?

Reply:
I don't have a big endian PPC system where I can verify the exact target triple value. That's why I've explicitly checked for little endian which I work on. It may be same for big endian too but not really sure. Even earlier, the fixes I pushed for ppc64le, had these checks.

Reply:
Yea, we only support TensorFlow on ppc64le (LE) in the community builds.